sp/src/game/server/basecombatcharacter.cpp

```diff
@@ -1646,6 +1655,15 @@ void CBaseCombatCharacter::FixupBurningServerRagdoll( CBaseEntity *pRagdoll )
 	}
 }
 
+inline bool CBaseCombatCharacter::ShouldFadeServerRagdolls() const
+{
+#ifdef MAPBASE
+	return IsNPC() ? HasSpawnFlags( SF_NPC_FADE_CORPSE ) : true;
```

Comment:
I should have left this as a comment on whatever the original PR to Mapbase was, but I wanted to question this a little. Why should this value always be true if not an NPC? What about prop_ragdolls?

Reply:
`ShouldFadeServerRagdolls` is only used to replace instances of where the value was already `true`. It only counts NPCs as an exception because only NPCs use this spawnflag, although I suppose it wouldn't be impossible to give it to players as well. That could be a feature for another time.

`prop_ragdoll`s spawn independently of `CBaseCombatCharacter` and do not use this.